From 853516f24f3c0ecf3a37a4bd11d37c3bb967b65e Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Fri, 20 Jul 2012 13:51:41 -0500
Subject: ENGR00217857: Changed iomux ID pin

Changed iomux MX6Q ID pin to MX6Q_PAD_ENET_RX_ER__ANATOP_USBOTG_ID
This fix was already implemented on CR ENGR00180424. Somehow this
was not included on newer releases.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.h b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
index e7f66e1..da6e438 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
@@ -179,7 +179,7 @@ static iomux_v3_cfg_t mx6q_sabreauto_pads[] = {
 	MX6Q_PAD_EIM_EB0__GPIO_2_28,
 
 	/* USBOTG ID pin */
-	MX6Q_PAD_ENET_RX_ER__ENET_RX_ER,
+	MX6Q_PAD_ENET_RX_ER__ANATOP_USBOTG_ID,
 
 	/* VIDEO adv7180 INTRQ */
 	MX6Q_PAD_ENET_RXD0__GPIO_1_27,
-- 
1.7.9.5

